core/src/test/java/org/apache/sedona/core/rangeJudgement/RangeFilterTest.java

```diff
@@ -0,0 +1,161 @@
+package org.apache.sedona.core.rangeJudgement;
```

Comment:
Could you please remove this test case because I have moved this test to somewhere else since your last PR. Thank you!

Reply:
Thanks for your reminder, I have deleted the test case RangeFilterTest.java from the current pull request